app/models/artefact.rb

```diff
@@ -344,7 +344,7 @@ def validate_redirect_url
 
   def valid_redirect_url_path?(target)
     URI.parse(target)
-    target.starts_with?("/") && target !~ %r{//} && target !~ %r{./\z}
+    target !~ %r{[^:]//}
```

Comment:
My understanding of the previous one would have been to ensure that it didn't have a "//" as that could have been to a different host e.g "//yahoo.com/test" but then it's also a bit strange that it checks that the target doesn't end with a "/" which seems purely stylistic.

Do you think it even needs a check? would we be expecting this to be validated by schemas anyway?